rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -117,6 +123,21 @@ def to_exported_dict(
                 exported_dict['asset'] = ''
             if tx_hash is not None:
                 exported_dict['notes'] = f'{evm_explorer}{tx_hash}  ->  {self.notes}'
+            if self.location in EVM_LOCATIONS and database is not None:
+                def _maybe_add_label_with_address(matched_address: re.Match[str]) -> str:
+                    """Adds a label of the address if it exists in a given string"""
```

Comment:
insted of defining it here each time maybe define it in the file as a private method. Otherwise you are creating the function each time